resources/metadata/detectExecuteScan.yaml

```diff
@@ -297,22 +297,11 @@ spec:
           - STAGES
           - STEPS
         deprecated: true
-      - name: useDetect8
+      - name: useDetect7
```

Comment:
You could also have set this to true by default. This avoids a breaking change due to parameter naming.

Reply:
Thanks for the note! Returned old parameters as deprecated